### PR TITLE
Use DOMjudge API to fetch images and team names for domlogo

### PR DESCRIPTION
**Pull request by nickygerritsen**

### CODE REVIEW

File: domlogo/domlogo.py [hunk 1]
```diff
@@ -50,6 +74,73 @@
...
+print('Downloading any new or changed logos and photos...')
+for organization in organizations.values():
+    if 'logo' in organization:
+        organization_id = organization['id']
+        logo = organization['logo'][0]
+        downloaded, downloaded_to, etag_file, etag = download_image('logo', organization_id, logo)
+        if downloaded_to:
+            # Convert to both 64x64 (for sidebar) and 160x160 (for overlay over photo)
+            target = f'domlogo-files/logos/{organization_id}.png'
+            command = f'convert {downloaded_to} -resize 64x64 -background none -gravity center -extent 64x64 {target}'
```
Comment (meisterT) [inline domlogo/domlogo.py:94]: I guess some shell escaping is prudent here?

Comment (nickygerritsen) [reply domlogo/domlogo.py:94]: We should have control over all variables but if you feel safer to escape, I can change it. Do you suggest to use `subprocess.run` for the whole command (like below) or to use `shlex.quote` to escape specific variables?

Comment (meisterT) [reply domlogo/domlogo.py:94]: shlex.quote should be fine here

File: domlogo/domlogo.py [hunk 2]
```diff
@@ -50,6 +75,74 @@
         break
 print(f'Using {api_url} as endpoint.')
 
+print('Loading teams and organizations from API')
+teams = {team['id']: team for team in requests.get(f'{api_url}/teams', auth=(user, passwd)).json()}
+for team_id in teams:
+    if 'display_name' not in teams[team_id]:
+        teams[team_id]['display_name'] = teams[team_id]['name']
+organizations = {org['id']: org for org in requests.get(f'{api_url}/organizations', auth=(user, passwd)).json()}
+
```
Comment (vmcj) [inline domlogo/domlogo.py:84]:
```suggestion
```

**Review by vmcj — APPROVED** [review]

**Review by meisterT — APPROVED** [review]

**Pull request merged**